### PR TITLE
Spelling, grammar, and clarity fixes

### PR DESCRIPTION
**Pull request by 6J7KZg2f**
Fixed all obvious errors, reworded some janky phrases, added missing translations, etc.

### CODE REVIEW

File: PiTool/pitool_en.ts [hunk 1]
```diff
@@ -263,7 +263,7 @@
     </message>
 </context>
 <context>
-    <name>ConnectModeSwicth</name>
+    <name>ConnectModeSwitch</name>
```
Comment (eras) [inline PiTool/pitool_en.ts]: This label may need to correlate with all the other mentions of it, so better change them all (including renaming files) or leave it alone?

**Review by eras — COMMENTED** [review]
Just checked this out on the morning so this is only as far I got, very nice changes :).

Did consider if it should be "switch Pimax mode" or "switch to Pimax mode" but I guess this form is fine as well.

:+1:

**Comment by peterhudson** [comment]
Reviewed phrasing changes and they look great. Thanks for going through and suggesting these changes. As a native English speaker, I can say this will be a huge improvement to the level of polish in the PiTool UI.

File: PiTool/pitool_en.ts (hunk 1 above)
Comment (6J7KZg2f) [reply PiTool/pitool_en.ts]: You're quite right, I'll revert that.

**Comment by 6J7KZg2f** [comment]
> Just checked this out on the morning so this is only as far I got, very nice changes :).
> 
> Did consider if it should be "switch Pimax mode" or "switch to Pimax mode" but I guess this form is fine as well.
> 
> 👍

I wasn't actually entirely sure about the context of that (and I did these translations by taking the centerpoint between google chinese -> japanese and google chinese -> english, because I don't speak Chinese :D )

**Pull request merged**